docs/reference/filters.md

```diff
@@ -471,6 +471,7 @@ Examples:
 
 ```
 oauthTokeninfoAnyKV("k1", "v1", "k2", "v2")
```

Comment:
The difference is very difficult to see when the two lines look that similar. Let's use on example line with both or change the example values to make this more visible.